cmd/piper.go

```diff
@@ -47,6 +47,7 @@ type GeneralConfigOptions struct {
 	GCSFolderPath        string
 	GCSBucketId          string
 	GCSSubFolder         string
+	URLReport            bool
```

Comment:
what do you think about using a `string` with the filename instead of a `bool` to activate it?
Then it would not require follow-up actions in case the target file needs to be customized.

Reply:
Let's also move the flag into the command package and not expose it via the `piper` command